src/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -3,14 +3,21 @@ import { Spinner } from 'components';
 import styles from './LoadingIndicator.module.scss';
 
 type LoadingIndicatorProps = {
-	style?: React.CSSProperties;
 	text: string | React.ReactNode;
+	subtext?: string | React.ReactNode;
+	style?: React.CSSProperties;
 };
 
 const LoadingIndicator = (props: LoadingIndicatorProps) => {
 	return (
 		<div style={props.style} className={styles.Container}>
 			<p>{props.text}</p>
+			{props.subtext && (
+				<>
+					<br />
+					<p>{props.subtext}</p>
+				</>
+			)}
```

Comment:
@domw30 Can we prevent `<p>`, `<br />` in JSX? I think we can use `<div />` and `classNames`.